### ISSUE TITLE
Please make a new release

### ISSUE THREAD

**Issue by mgorny** [ahupp/python-magic#309]
Could you please make a new release, since 0.4.27 fails tests with the current version of `file(1)`?

**Comment by ahupp** [comment]
Yes, I have a pile of changes I was hoping to make before the next release but I might do a little one first beforehand.

**Comment by baraknaveh** [comment]
Is there a rough ETA for a new release?
